apps/web/src/api/hooks/useUpdateOrganizationBranding.tsx

```diff
@@ -19,8 +21,11 @@ export const useUpdateOrganizationBranding = (
   const { mutateAsync: updateOrganizationBranding, isLoading } = useMutation<ResultType, IResponseError, PayloadType>(
     (payload) =>
       updateBrandingSettings({
-        logo: payload.logoUrl,
-        color: payload.colorValue,
+        logo: payload.brandLogoUrl || undefined,
```

Comment:
**question:** Is it possible to align this object with the API payload to avoid this transformation? If we own both ends of the APIs and there are no backward compatibility concerns I suggest we do it. ‏